index.html

```diff
@@ -4,26 +4,146 @@
+          <li class="list-item2">Sponsors</li>
+          <li class="list-item2">News</li>
+          <li class="list-item2b">Presentation</li>
+        </ul>
+      </nav>
+    </div>
   </header>
+  <h1 class="not-appear">FICA Soccer Tournament</h1>
+  <section class="section1-wrapper">
+    <h2 class="not-appear">FICA Soccer Tournament</h2>
+    <div class="section1">
+      <img id="title" src="img/title.png" width=324 height=110 alt="FICA 2022 Tournament">
+      <p class="text-box"> A fun festival where teams from 10 countries who believe in the value
+        of friendship and sports gather and create positive change, which will be held in Buenos Aires in September.
+      </p>
+      <p class="schedule"></p><div class="sched-text">2022.09.14 (Wed) ~ 18 (Sun)</div>
+        <div class="big">@ River Plate Stadium of Buenos Aires, Belgrano, etc.</div><p></p>
+    </div>
+  </section>
 
-  <section>
+  <section class="section2-wrapper">
+    <div class="section2">
+      <h2 class="myh1">Main Program</h2>
+      <hr class="hr2">
+      <div class="programs-wrapper">
+        <div class="program-block">
+          <img src="img/program_icon_01.png" width=64 height=58 alt="Presentation">
+          <div class="subtitle">Presentation</div>
+          <div class="description">The 10 teams will be presented together with their background and records, and key players.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_02.png" width=64 height=58 alt="The Games">
+          <div class="subtitle">The Games</div>
+          <div class="description">The 16 games divided into four days, with a minimum of four games per team will be played.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_03.png" width=64 height=58 alt="Press conference">
+          <div class="subtitle">Conference</div>
+          <div class="description">After each game two team members, one of them being the coach, will answer questions to the press.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_04.png" width=64 height=58 alt="Dissertation">
+          <div class="subtitle">Dissertation</div>
+          <div class="description">The closing day a dissertation by Marcelo Bielsa will precede the tournament game.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_05.png" width=64 height=58 alt="Awards">
+          <div class="subtitle">Awards</div>
+          <div class="description">After the closing game, the award ceremony will take place with a small convite for the participants and guests.</div>
+        </div>
+      </div>
+      <div class="texttoright">View Full Program ></div>
+    </div>
+  </section>
 
+  <section class="section3-wrapper">
+    <div class="section3">
+      <h2 class="myh1b">main teams</h2>
+      <hr class="hr2">
+      <div class="teams-wrapper">
+      </div>
```

Comment:
Great work, implementing the required design. 💯 

[OPTIONAL] - You've made a great effort to align the text to the right. However, it is still wrapped under the image. It would look nicer if was completely on the right. Otherwise, fantastic job!!! 👏🏾 

![image](https://user-images.githubusercontent.com/38088818/163626339-9e48a58b-2f81-472b-ac31-3b3218497cb1.png)

Reply:
This requires a re-design because there's little room for all the text. I'll see if I can accommodate it.